server/model/monitor.js

```diff
@@ -100,6 +100,12 @@ class Monitor extends BeanModel {
             mqttSuccessMessage: this.mqttSuccessMessage,
             databaseQuery: this.databaseQuery,
             authMethod: this.authMethod,
+            authWorkstation: this.authWorkstation,
+            authDomain: this.authDomain,
```

Comment:
103-104 do not seem to belong this pr, it is a merge issue?

Reply:
Yes, it looks like those lines are inserted after merged from master.